locales/ku/LC_MESSAGES/duckduckgo.po

```diff
@@ -5446,28 +5470,28 @@ msgstr ""
 #. Label for tips tab
 msgctxt "Covid 19 module"
 msgid "Tips"
-msgstr ""
+msgstr "Şîret"
 
 msgid "Tired of being tracked online? %sWe can help.%s"
-msgstr ""
+msgstr "Tu westiya ye ji şopandina serhêl? %sEm dikarin alîkariyê bikin.%s"
 
 #. This is the name of a user setting
 msgctxt "settings"
 msgid "Title Color"
-msgstr ""
+msgstr "Rengê Sernavê"
 
 #. This is the name of a user setting
 msgctxt "settings"
 msgid "Title Font"
-msgstr ""
+msgstr "Curenivîsa Sernavê"
 
 #. This is the name of a user setting
 msgctxt "settings"
 msgid "Title Underline"
-msgstr ""
+msgstr "Binxêzkirina Sernavê"
 
 msgctxt "directions"
-msgid ""
+msgid "Rêgeh"
 "To print directions:%sGo back to the map.%sSelect the route you want to "
 "print.%sClick the print icon.%s"
 msgstr ""
```

Comment:
Watch out, this is wrong. You translated the `msgctxt` which is not user-visible. The actual text to translate is `To print directions...`. Please revert this. Thank you 🙂 .

Reply:
You could also create a POEditor.com project for this.